docs/src/modules/components/ApiPage.js

```diff
@@ -284,6 +284,8 @@ import { ${componentName} } from '${source}';`}
           </React.Fragment>
         )}
         <Heading hash="props" />
+        <span dangerouslySetInnerHTML={{ __html: spreadHint }} />
```

Comment:
Could we add more verticale spacing between this sentence and the table? It feels not spaced enough to create a clear isolation (two different pieces of information).

Reply:
We could sure. I wouldn't do that since these are coupled together. There's already enough space.

Reply:
Yeah, having a closer look, we can discard my comment, my concern was more with the consistency, I cover it better in: https://github.com/mui-org/material-ui/pull/26778/files#r654036156.